baseframe/utils.py

```diff
@@ -0,0 +1,59 @@
+import six
+from six.moves.urllib.parse import quote as urlquote
+from . import asset_cache
+from mxsniff import mxsniff, MXLookupException
```

Comment:
`mxsniff` is an external dependency and must be imported after `six` and before `.`